src/test/scala/latis/input/HapiJsonAdapterSpec.scala

```diff
@@ -10,7 +10,7 @@ import latis.util.FileUtils
 import org.scalatest.FlatSpec
 import org.scalatest.Matchers._
 
-class TestHapiJsonAdapter extends FlatSpec {
+class HapiJsonAdapterSpec extends FlatSpec {
```

Comment:
Let's hold off on this change. I'm refactoring the json adapter to use the base `HapiAdapter` query building stuff (and factoring out your stuff to a more generic `JsonArrayAdapter`). I'll then look into issues reading data from the hapi server. I'll make `HapiJsonAdapterSpec` in my branch.

Reply:
Okay, I'll change `HapiJsonAdapterSpec` back to `TestHapiJsonAdapter`.